addons/fluxcd/definitions/kustomize.cue

```diff
@@ -82,6 +85,20 @@ template: {
 						_sourceCommonArgs
 					}
 				}
+				if parameter.repoType == "oci" {
```

Comment:
```suggestion
				if parameter.repoType == "oci" && parameter.oci != _|_ {
```

Reply:
I think the `parameter.oci` can be empty, right?

Reply:
Yes this parameter.oci can be empty